runtime/cmd/decode-state-values/main.go

```diff
@@ -258,111 +259,137 @@ func load() {
 
 	bar := progressbar.Default(int64(len(storage)))
 
+	var slabNotFoundErrCount int
+
 	for storageKey, data := range storage { //nolint:maprangecheck
 		_ = bar.Add(1)
 
-		var isStoragePath bool
-
 		// Check the key is a non-root slab or a storage path
 		key := storageKey[2]
 
 		var address atree.Address
 		copy(address[:], storageKey[0])
 
-		// If the key is for a slab (format '$' + storage index),
-		// then attempt to decode the slab
+		err := loadStorageKey(key, address, data, inter, slabStorage)
+		var slabNotFoundErr *atree.SlabNotFoundError
+		if errors.As(err, &slabNotFoundErr) {
+			slabNotFoundErrCount++
+		}
```

Comment:
I think `atree.SlabNotFoundError` might not be returned from `loadStorageKey`.

Reply:
`loadStorageKey` calls `StoredValue` on the decoded slab, which in turn might more slabs. The decoded state might be corrupt, which might cause atree to load a slab which does not exist, resulting in a `SlabNotFoundError`